tests/unit/api/test_offline_ingest_query_graph.py

```diff
@@ -440,6 +440,39 @@ def odfv_feature_zscore_v231227_{feature_model.id}(
     """
     assert offline_store_info.odfv_info.codes.strip() == textwrap.dedent(expected).strip()
 
+    feat = feature_raw[feature_avg > 0]
+    feat.name = "feature"
```

Comment:
I think a more common scenario is a conditional assign operation on feature. Would that involve FilterNode? Wondering if that is a special case that should also be covered by a test, or maybe this test is sufficient.

Reply:
Conditional assign node does not involve filter node. That's why initially i thought it should not be support. This test will trigger filter node code generation as shown in the generated code below.